tiup-mirror: Add a hint about how to use a tiup mirror (#5336)
pingcap/docs: tiup/tiup-mirror.md

This section introduces the usage examples of the `mirror` command.

> The repository that was cloned with `tiup mirror clone` can be shared among hosts either by sharing the files via SCP, NFS, etc or by making the repository available over HTTP or HTTPS. Use `tiup mirror set <location>` to specify location of the repository.

Reviewer: ```suggestion
The repository that is cloned using `tiup mirror clone` can be shared among hosts either by sharing the files via SCP, NFS or by making the repository available over the HTTP or HTTPS protocol. Use `tiup mirror set <location>` to specify the location of the repository.
```
